documentation/modules/snippet_secret-cr.adoc

```diff
@@ -0,0 +1,65 @@
+:_content-type: SNIPPET
+
+
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
```

Comment:
@ahadas unrelated, but it's very weird the user needs to set an `ownerReference`

Reply:
@bennyz see the description below ("The `ownerReferences` section is optional") - the user can skip that if they don't mind about cleanup